src/fixit/ftypes.py

```diff
@@ -46,8 +46,8 @@
 
 NodeReplacement = Union[CSTNodeT, FlattenSentinel[CSTNodeT], RemovalSentinel]
 
-Timings = Dict[str, int]
-TimingsHook = Callable[[Timings], None]
+LoggableMetric = Dict[str, Any]
+LoggerHook = Callable[[LoggableMetric], None]
```

Comment:
I think I'd personally avoid the "logger" nomenclature so as to avoid confusion with the builtin `logging` module/etc.  `Metrics` and `MetricsHook` would be my suggestions for these two type aliases.  Similarly I'd suggest using `metrics` and `metrics_hook` for variable names elsewhere.